ajax/libs/xdomain/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "xdomain",
+  "filename": "xdomain.min.js",
+  "version": "0.7.3",
+  "homepage": "https://github.com/jpillora/xdomain",
+  "authors": "Jaime Pillora <dev@jpillora.com>",
+  "description": "A pure JavaScript CORS alternative",
+  "keywords": [
+    "cors",
+    "ie8",
+    "polyfill"
+  ],
+  "license": "MIT",
+  "repositories": [
+    {
+      "type": "git",
+      "url": "https://github.com/jpillora/xdomain"
+    }
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/jpillora/xdomain.git",
+    "basePath": "dist/",
+    "files": [
+      "xdomain*.js"
```

Comment:
Did you mean that I should use `xdomain*.*` here? I am a little confuse.
I use `xdomain*.js` here because there are old version exist in `/dist/0.6/`.